azext_iot/_help.py

```diff
@@ -1340,6 +1340,53 @@
     short-summary: Delete an enrollment group in an Azure IoT Hub Device Provisioning Service.
 """
 
+helps[
+    "iot dps enrollment-group registration"
+] = """
+    type: group
+    short-summary: Manage device registrations for an enrollment group in an Azure IoT Hub Device
+        Provisioning Service.
+"""
+
+helps[
+    "iot dps enrollment-group registration list"
+] = """
+    type: command
+    short-summary: List device registrations for an enrollment group in an Azure IoT Hub Device
+        Provisioning Service.
+"""
+
+helps[
+    "iot dps enrollment-group registration show"
+] = """
+    type: command
+    short-summary: Get a device registration for an enrollment group in an Azure IoT Hub Device
+        Provisioning Service.
+"""
+
+helps[
+    "iot dps enrollment-group registration delete"
+] = """
+    type: command
+    short-summary: Delete a device registration in an Azure IoT Hub Device Provisioning Service.
+"""
+
+helps[
+    "iot dps enrollment-group compute-device-key"
+] = """
+    type: command
+    short-summary: Generate a derived device SAS key.
+    long-summary: Generate a derived device key for a DPS enrollment group.
```

Comment:
We can eliminate `long-summary` here and combine the bit of additional context with `short-summary`.